client/extensions/wp-job-manager/components/setup/page-setup/index.jsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { formValueSelector, reduxForm } from 'redux-form';
+import { localize } from 'i18n-calypso';
+import { flowRight as compose } from 'lodash';
```

Comment:
May I ask why the fancy name 😄  ?

Reply:
I've seen it done [elsewhere](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=flowRight+as+compose&type=) in the codebase and it seems to be a more appropriate name to describe what it does. So thought I'd adopt it. 😃